rise/lib/services/categories.ts

```diff
@@ -17,3 +17,18 @@ export const getCategories: QueryResolvers['categories'] = async (
   `
   )
 }
+
+export async function getCategoryById(categoryId: string): Promise<any[]> {
```

Comment:
Perhaps this is more appropriate? It would force us to handle the null-cases throughout the code-base as well!
```suggestion
export async function getCategoryById(categoryId: string): Promise<Category | null> {
```